packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js

```diff
@@ -64,7 +64,16 @@ angular.module('App').controller(
 
                 // add user friendly translations for some known tasks
                 if (
-                  indexOf(['action', 'memberContactXXX'], argument.key) !== -1
+                  indexOf(
+                    [
+                      'action',
+                      'memberContactXXX',
+                      'firstname',
+                      'name',
+                      'identificationNumber',
+                    ],
+                    argument.key,
+                  ) !== -1
```

Comment:
**nitpick** : we could've turn it into an `includes`

Reply:
Thanks, suggestion has been applied.